app/_landing_pages/gateway/deployment-topologies.yaml

```diff
@@ -8,12 +8,224 @@ rows:
+            description: |
+              Learn about self-hosting Data Plane nodes.
+              
+            icon: /assets/icons/self-hosted.svg
+            cta:
+              url: "/gateway/topology-hosting-options/#self-hosted"
+      
+  - header:
+      type: h2
+      text: "DB-less mode"
+    columns:
+      - blocks:
+          - type: mermaid
+            config:
+              diagram: |
+                flowchart TD
+                  A(<img src="/assets/logos/KogoBlue.svg" style="max-height:20px"> Kong Gateway instance)
+
+                  A2(fa:fa-file kong1.yml)
+
+                  A2 --> A
+
+                  style A stroke:none,fill:#0E44A2,color:#fff
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: |
+                    You can enable DB-less mode to reduce complexity of and create more flexible deployment patterns. In this mode, configured entities such as Routes, Services and plugins are stored in-memory on the node.
+                    When running in DB-less mode, configuration is provided to {{ site.base_gateway }} using a second file. This file contains your configuration in YAML or JSON format using {{site.base_gateway}}'s declarative configuration syntax.
+
+                    The combination of DB-less mode and declarative configuration has a number
+                    of benefits:
+
+                    * Reduced number of dependencies: no need to manage a database installation
+                      if the entire setup for your use-case fits in memory.
+                    * Your configuration is always in a known state. There is no intermediate 
+                      state between creating a Service and a Route using the Admin API.
+                    * It is a good fit for automation in CI/CD scenarios. Configuration for
+                      entities can be kept in a single source of truth managed via a Git
+                      repository.
+
+                    Here are a few limitations of this mode:
+
+                    * The [Admin API](/api/gateway/admin-ee/) is read only.
+                    * Any plugin that stores information in the database, like rate limiting (cluster mode), don't fully function.
+
+                    **Works on:** {{site.base_gateway}}
+          - type: button
+            config:
+              align: left
+              text: Learn more about DB-less mode
+              url: /gateway/db-less-mode/
+  - header:
+      type: h3
+      text: "Supported Data Plane hosting options"
+    columns: 
+      - blocks:
+        - type: card
+          config:
+            title: Self-hosted Data Plane nodes
+            description: |
+              Learn about self-hosting Data Plane nodes.
+              
+            icon: /assets/icons/self-hosted.svg
+            cta:
+              url: "/gateway/topology-hosting-options/#self-hosted"
```

Comment:
This feels hidden at the bottom. Can we try it at the top at the right hand side (alongside the "When you deploy Kong Gateway, you can choose how your Data Plane nodes are hosted based on the [deployment topology](https://deploy-preview-812--kongdeveloper.netlify.app/gateway/deployment-topologies/) you’re using." text)with a description?

Something like:

> Looking for ways to host your Data Plane? See the _Supported Data Plane hosting options_